docs/installation/Setup-OpenFunction-Builder-with-Tekton.md

```diff
@@ -0,0 +1,154 @@
+# Setup OpenFunction Builder with Tekton
+
+You can refer to the installation steps on the [Tekon docs](https://tekton.dev/docs/getting-started/) for setup OpenFunction Builder. Or follow these steps:
```

Comment:
=>
You can refer to the steps in [Tekton docs](https://tekton.dev/docs/getting-started/) to install Tekton or follow the steps below: